ocw-cli/cli_app.py

```diff
@@ -599,123 +608,56 @@ def run_screen(model_datasets, models_info, observations_info,
 
              screen.addstr(6, 4, "Spatially regridding...")
              screen.refresh()
-             new_lats = np.arange(overlap_min_lat, overlap_max_lat, spatial_grid_setting)
-             new_lons = np.arange(overlap_min_lon, overlap_max_lon, spatial_grid_setting)
-             for i in range(len(obs_dataset)):
-                  obs_dataset[i] = dsp.spatial_regrid(obs_dataset[i], new_lats, new_lons)
+             if spatial_grid_option == 'Observation':
```

Comment:
No need for this "if" condition. 
Checkout line 1241 on file https://github.com/apache/climate/blob/master/ocw-cli/cli_app.py